crates/valence_protocol_macros/src/ident_str.rs

```diff
@@ -36,6 +36,6 @@ pub fn ident_str(item: TokenStream) -> Result<TokenStream> {
     };
 
     Ok(quote! {
-        unsafe{Ident::new_unchecked(#ident, #path_start)}
+        Ident::new_unchecked(#ident, #path_start)
```

Comment:
`Ident` needs to be fully qualified because it might not be in scope where the macro is used.

```rust
::valence_protocol::ident::Ident::new_unchecked(#ident, #path_start)
```